gem/lib/metasploit-payloads/crypto.rb

```diff
@@ -0,0 +1,66 @@
+require 'openssl'
+
+module MetasploitPayloads
+  module Crypto
+    CIPHERS = {
+      chacha20: {
+        name: 'chacha20'.b,
+        version: 1,
+        iv: {
+          value: 'EncryptedPayload'.b, # 16 bytes
```

Comment:
let's just generated something random here

```
3.0.5 :006 > require 'securerandom'
 => false 
3.0.5 :015 > puts SecureRandom.random_bytes(16).bytes.map { |x| "\\x#{x.to_s(16)}" }.join
\x69\xc4\x69\x80\xe5\x26\x36\xd7\x6e\x16\x38\x22\x6c\xcc\xf4\xc4
 => nil 
```

(I'm sure a pack directive could've done that for us too)